src/DigitalTwin/CodeGeneratorCore.ts

```diff
@@ -23,7 +23,7 @@ import {RemoteExtension} from '../Models/RemoteExtension';
 import {DigitalTwinUtility} from './DigitalTwinUtility';
 import {CodeGenUtility} from './DigitalTwinCodeGen/CodeGenUtility';
 import {FileUtility} from '../FileUtility';
-import {CancelOperationError} from '../CancelOperationError';
+import {CancelOperationError} from '../common/CancelOperationError';

```

Comment:
It is good to move this to common folder, and I think it will be better to put all Error class definition in one file, so can we rename it for general purpose like Errors.ts?

Reply:
there is a typo in CancelOperationError class: "Openration cancelled" -> "Operation cancelled". Can you help fix this in this PR as well?

Reply:
The typo is fixed in `pre-release-0.11.0-rc` branch (see [commit](https://github.com/microsoft/vscode-iot-workbench/commit/529944efd1374d493d0c66d5d01d63adcb192525#diff-98676cbf9c6148ad805552a7eed6c0c3)). Later this branch will be merged back to develop branch.

I will double check this typo after the prerelease branch merge back into develop branch.

Reply:
Currently we have only one special error type: the `CancelOperationError`. If we have more error types, I will make this file a more general one.

Reply:
Will do that after move to eslint.